### ISSUE TITLE
[Enhancement] Rotate the top/bottom UVs for panel stairs to match panel blocks & slabs

### ISSUE THREAD

**Issue by triggerhurt** [Yamahari/ILikeWood-1.15#49]
The current panel stair model creates a minor inconsistency in how stairs, slabs, and planks interact when used in combination, which causes them to stand out a bit:
![image](https://user-images.githubusercontent.com/32649483/154349054-19ed797a-6779-46c3-9569-05b4b6cf659f.png)

An example of how it looks in vanilla:
![image](https://user-images.githubusercontent.com/32649483/154349337-e01d42e6-4012-42c6-858c-4e4eaeb682af.png)

**Comment by triggerhurt** [comment]
Ah. I've just realized that an issue exists for this already (issue #8). My apologies.

**Comment by Yamahari** [comment]
Yup
This has been there since the beginning, might need to change the way panels look to fix it properly
Probably needs to be a horizontally rotatable planks block turned on it's side x)

**Comment by triggerhurt** [comment]
Here, I fixed it
![image](https://user-images.githubusercontent.com/32649483/154360440-3e305ce3-a36a-4df2-8f86-e4380195b909.png)
![image](https://user-images.githubusercontent.com/32649483/154360611-7533eaa7-84d9-4db0-a447-e28e59244568.png)
![image](https://user-images.githubusercontent.com/32649483/154361001-93e9e297-9fae-4c92-892a-c17b29837547.png)

This is a resourcepack that edits the three template files for the stair blockmodels; I don't know how pull requests work or if I can even create one, but feel free to use it in the mod, if possible.
[ilikewood-stairfix.zip](https://github.com/Yamahari/ILikeWood/files/8083732/ilikewood-stairfix.zip)

**Comment by Yamahari** [comment]
Does it work if you turn the stairs upside down though

**Comment by triggerhurt** [comment]
Yep. The first screenshot demonstrates all stair variants in action
Edit to add a screenshot showing the upside-down stairs working
![image](https://user-images.githubusercontent.com/32649483/154372558-6aac35ee-cdad-4cf7-83b2-08d1a241c111.png)

**Comment by Yamahari** [comment]
![2022-02-17_09 58 34](https://user-images.githubusercontent.com/40495288/154441522-f94c74c2-9440-4087-9706-bc86de7d6f3b.png)
![2022-02-17_09 59 23](https://user-images.githubusercontent.com/40495288/154441528-80a12ac5-90ad-49be-b7a8-9b03925b21a6.png)
![2022-02-17_09 57 12](https://user-images.githubusercontent.com/40495288/154441537-69d93e45-a1ab-4ead-9b4d-a741dbd72d67.png)
![2022-02-17_09 57 39](https://user-images.githubusercontent.com/40495288/154441543-e86b2156-811c-44d7-9fbb-39494c138587.png)
![2022-02-17_09 58 03](https://user-images.githubusercontent.com/40495288/154441551-d02c30be-bd1d-4bff-baf0-b85da6a0c6b3.png)

still doesnt line up on some rotations properly x) I will have a look at it when I am doing with what I am working on atm

**Comment by triggerhurt** [comment]
Seems I didn't realize the scope of the problem: Apparently it's an issue with Minecraft's stair block implementation (or possibly general rotations...); evidently it doesn't really like rotated UVs. I wish you the best of luck on coming up with a good solution.
It's kinda weird that it doesn't have a problem with upside-down stairs, it just specifically breaks the uv orientation for regular stairs, despite using the exact same blockmodel...
